app/js/components/header/View.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import '../../../css/header.css';
+
+const View = props => (
+  <header>
+    <div className="logo">
+      <a href="../../referenceapplication/home.page">
+        <img src="../../ms/uiframework/resource/uicommons/images/logo/openmrs-with-title-small.png" alt="" />
+      </a>
+    </div>
+
+    <ul className="user-options">
+      <li
+        role="button"
+        tabIndex="0"
+        className="identifier"
+        style={{ cursor: "pointer" }}
+        onClick={() => { props.toggleState("userDropdown"); }}
+      >
+        <i className="icon-user small" />
+        {props.currentUser}
+
+        <i className="icon-caret-down appui-icon-caret-down link" /><i className="icon-caret-up link appui-toggle" style={{ display: "none" }} />
+        {props.userDropdown &&
+          <ul id="user-account-menu">
+            <li>
+              <a
+                href="'../../../../adminui/myaccount/myAccount.page">Account
+              </a>
+            </li>
+          </ul>
+        }
+
+      </li>
+      <li className="change-location">
+        <a href="#" onClick={() => { props.toggleState("locationDropdown"); }}>
+          <i className="icon-map-marker small" />
+          <span id="selected-location">{props.currentLocation.display}</span>
+
+          <i className="link icon-caret-down" />
+        </a>
+        {props.locationDropdown &&
+          <div>
+            <ul className="location-container">
+              {props.locations.map(location => (
+                <li
+                  role="button"
+                  key={location.uuid}
+                  onClick={() => {
+                    props.setCurrentLocation(location.uuid);
+                    props.toggleState("locationDropdown", false);
+                  }}
+                  className={location.display === props.currentLocation.display ? "selected" : ""}
+                >
+                  {location.display}
+                </li>
+              ))}
+            </ul>
+          </div>
+        }
+      </li>
+      <li className="logout">
+        <a href="/openmrs-standalone/appui/header/logout.action?successUrl=openmrs-standalone">
```

Comment:
Are you sure everyone will use the standalone version?

Reply:
Oh my, the logic was to make the url dynamic, sorry I forgot this one, will fix it now

Reply:
Done, this has been fixed in a couple of places too